portality/tasks/ingestarticles.py

```diff
@@ -562,7 +558,10 @@ def __fail(record, previous, error):
             return __fail(record, previous, error="please check it before submitting again; " + str(e))
 
 
-@main_queue.task(**configure("ingest_articles"))
+huey_helper = IngestArticlesBackgroundTask.create_huey_helper(main_queue)
+
+
+@huey_helper.register_execute(is_load_config=True)
 @write_required(script=True)
```

Comment:
I can see `register_execute` is also calling `write_required` so can we remove it from here?

Reply:
yes, it should be remove.